examples/export-terraform-imperative/Kptfile

```diff
@@ -0,0 +1,5 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: export-terraform-imperative
+pipeline: {}
```

Comment:
Is this file necessary?

Reply:
Yes, I think so. The e2e tests failed when I didn't have a Kptfile.